README.md

```diff
@@ -131,6 +131,11 @@ To the extent possible under law, all contributors have waived all copyright and
 
 - [Dados abertos do Rio Grande do Sul](http://dados.rs.gov.br) &mdash; Dados de orgãos púplicos estaduais.
 - [Dados Prefeitura de Porto Alegre](http://datapoa.com.br) &mdash; Dados de orgãos e entidades de administração pública municipal.
+- [Dados abertos TCE-RS](http://dados.tce.rs.gov.br/) &mdash; Dados abertos do tribunal de contas do estado.
+- [VisualizaDEE](http://visualiza.dee.planejamento.rs.gov.br/) &mdash; Aplicativos de visualização de dados do Departamento de Economia e Estatística do RS.
```

Comment:
Can you confirm that this website provides open data besides the visualizations?

Reply:
Yes. The visualizations use Shiny (https://shiny.rstudio.com/) and inside each one there is a link "Download dos Dados" with open data download.